contracts/usul/BaseQuorumPercent.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+
+/// @title BaseQuorumPercent - A Usul strategy extension that enables percent based quorums
+abstract contract BaseQuorumPercent is OwnableUpgradeable {
```

Comment:
why is this `OwnableUpgradeable` instead of just `Ownable`? Is that necessary here?

Reply:
So both `Owanble` and `OwnableUpgradeable` allows the `owner` to set a new `owner`.

The reason we need to use `OwnableUpgradeable` here is because this contract won't be set up using a constructor, but instead using an initializer function. `Ownable` has a constructor, which means that any contracts inheriting from it need to call that from their own constructor.